lib/WP_Auth0_UsersRepo.php

```diff
@@ -161,41 +161,51 @@ public function create( $userinfo, $token, $access_token = null, $role = null, $
 		return $user_id;
 	}
 
-	public function find_auth0_user( $id ) {
+	/**
+	 * Look for and return a user with an Auth0 ID
+	 *
+	 * @param string $id - An Auth0 user ID, like "provider|id"
+	 *
+	 * @return null|WP_User
+	 */
+	public function find_auth0_user ( $id ) {
```

Comment:
Just asking. Why is this method needed?

Reply:
@lbalmaceda - Finding the WP user with Auth0 info